packages/client/src/runtime/core/errorRendering/applyValidationError.test.ts

```diff
@@ -622,8 +776,6 @@ describe('UnknownSelectionField', () => {
         include: {
           notThere: true,
           ~~~~~~~~
-      ?   id?: true,
```

Comment:
Opprtunistic fix: we previously suggested to use scalar fields in `include`. Only relations are valid in include, so those suggestions were incorrect.